esi/src/parse.rs

```diff
@@ -106,7 +106,7 @@ where
     Ok(())
 }
 
-fn parse_include<'a, 'b>(elem: &'a BytesStart) -> Result<Event<'b>> {
+fn parse_include<'a>(elem: &BytesStart) -> Result<Event<'a>> {
```

Comment:
Can you explain this change? Why were there two lifetimes before?

Reply:
I think this is down to a change in `quick_xml` which means the compiler is now able to infer a lifetime for the `BytesStart`.